### ISSUE TITLE
Automate the "last updated" field

### ISSUE THREAD

**Issue by awfuchs** [GoogleChrome/developer.chrome.com#914]
⚠️ This is for feature requests with the https://developer.chrome.com site, not Chromium itself.
If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.

When making and update to the docs, it's easy to forget about updating the "updated: <date>" metadata.

It would be nice to have this automated in some way, so that I'm either reminded (say, on commit) or can use a shortcode like "updated: {% last-commit %}" or some similar mechanism. (To avoid revving this date on little typo fixes, perhaps an "updated: {% unchanged %}" code could work.) Overall, it would be nice to grab the date of merge to main rather than the last local commit, but that could work too.

**Comment by MichaelSolati** [comment]
I don't think we could automate adding/updating the update date, as I recall that added issues on web.dev. However, web.dev has a script `npm run updated` which will update the date of modified posts. Would something like that work?

See GoogleChrome/web.dev#3534

**Comment by awfuchs** [comment]
I don't see how `npm run updated` would be easier to remember than updating the date by hand... What kind of issues did automating this cause on web.dev?

Could we have a precommit trigger that prompts to update the date?

**Comment by MichaelSolati** [comment]
So the issue is that the precommit hook would modify other files and then would make it crazy difficult to correct. This issue probably better explains it than I can => GoogleChrome/web.dev#3368
